### PR TITLE
remove erc and fix workflow errors

### PR DESCRIPTION
**Pull request by Ja7ad**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -23,4 +23,4 @@ gem "wdm", "~> 0.1.1" if Gem.win_platform?
 
 gem "html-proofer", '>=5.0.7'
 
-gem "webrick", ">= 2.2.8"
+gem "webrick", '>= 1.8.1'
```
Comment (b00f) [inline Gemfile:26]: Why reduce the version?

File: rss/nonerc-last-call.xml [hunk 2]
```diff
@@ -11,7 +11,7 @@ layout: null
     <lastBuildDate>{{ site.time | date_to_rfc822 }}</lastBuildDate>
     {% assign pips = site.pages | sort: 'pip' %}
     {% for pip in pips %}
-      {% unless pip.category == "ERC" %}
+      {% unless pip.category == "PIP" %}
```
Comment (b00f) [inline rss/nonerc-last-call.xml:14]: Category can't be PIP, based on the above definition categories are: "Core, Networking, Interface"

File: rss/nonerc.xml [hunk 3]
```diff
@@ -10,7 +10,7 @@ layout: null
     <lastBuildDate>{{ site.time | date_to_rfc822 }}</lastBuildDate>
     {% assign pips = site.pages | sort: 'pip' %}
     {% for pip in pips %}
-      {% unless pip.category == "ERC" %}
+     {% unless pip.category == "PIP" %}
```
Comment (b00f) [inline rss/nonerc.xml:13]: Same here

File: Gemfile.lock [hunk 4]
```diff
@@ -2,8 +2,6 @@ GEM
   remote: https://rubygems.org/
   specs:
     Ascii85 (1.1.0)
-    activemodel (7.0.4.3)
```
Comment (b00f) [inline Gemfile.lock:5]: Any reason to remove it?

**Review by b00f — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (Ja7ad) [reply Gemfile:26]: @b00f latest is 1.8.1

https://rubygems.org/gems/webrick/

File: Gemfile.lock (hunk 4 above)
Comment (Ja7ad) [reply Gemfile.lock:5]: Nope check

**Comment by b00f** [comment]
try this:

```
gem "html-proofer", '>=4.0'

gem "webrick", '>= 1.8'
```

**Comment by Ja7ad** [comment]
> try this:
> 
> ```
> gem "html-proofer", '>=4.0'
> 
> gem "webrick", '>= 1.8'
> ```

```shell
The dependencies in your gemfile changed

You have added to the Gemfile:
* html-proofer (>= 4.0)
* webrick (>= 1.8)

You have deleted from the Gemfile:
* html-proofer (>= 5.0.7)
* webrick (>= 1.8.1)
Error: The process '/opt/hostedtoolcache/Ruby/3.2.2/x64/bin/bundle' failed with exit code 16
```

**Review by b00f — APPROVED** [review]

**Review by kehiy — APPROVED** [review]
LGTM

**Pull request merged**